build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -32,7 +32,15 @@
         <Link>app.config</Link>
       </None>
     </ItemGroup>
+  </Target>
 
+  <!-- Add Test Explorer Service GUID required by VS, and error out if it was specified in the project file -->
+  <!-- See https://connect.microsoft.com/VisualStudio/feedback/details/800245/vs2013rc-adds-to-vs2012-c-project-section-itemgroup -->
+  <Target Name="AddTestExplorerServiceGUID">
+    <Error Condition="'%(Service.Identity)' == '{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}'" Text="Test Explorer Service GUID exists in project. It is specified in Roslyn.Toolsets.Xunit.targets" />
+    <ItemGroup>
```

Comment:
It is added when the user debug/run tests from within VS (TestDriven.Net). So far, I've done manual testing and it is not added.
If I found another repro, I'll investigate it as well.

---
In reply to: [103040177](https://github.com/dotnet/roslyn/pull/17370#discussion_r103040177) [](ancestors = 103040177)